### PR TITLE
updated create project and create bug

### PR DESCRIPTION
**Pull request by sachind-cuelogic**
- Updated create project UI
- Updated create bug UI

### CODE REVIEW

File: bms/bms_app/static/bms_app/css/create_project.css [hunk 1]
```diff
@@ -118,28 +72,28 @@ label {
...
-	background-color:  #5093e1;
-	border: none;
-	color: #fff;
+	margin-right: 15px;
 }
 
 @media (max-width: 767px){
 	#user-role{
-		margin: 15px 0px;
+		margin: 15px 0px 15px 0px;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_project.css]: Repeated mistake..... Read about CSS short hand properties and implement it as per use case.

File: bms/bms_app/static/bms_app/css/layout.css [hunk 2]
```diff
@@ -160,7 +142,7 @@ header{
 .app-search {
     position: relative;
     margin: 10px 0px 0px 0px;
-    padding-left: 50px;
+    margin-right: 15px;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/layout.css]: unnecessary margin-right property added.

File: bms/templates/registration/header_sidebar.html [hunk 3]
```diff
@@ -16,95 +16,102 @@
...
-        {% block jsfile %} {% endblock%}
-    </body>
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script type='text/javascript' src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="{% static 'bms_app/js/sidebar.js' %}"></script>
+    <script src="https://cdn.datatables.net/1.10.15/js/jquery.dataTables.min.js"></script>
+    <script src="https://cdn.datatables.net/1.10.15/js/dataTables.bootstrap.min.js"></script>
+    {% block jsfile %} {% endblock%}
+</body>
```
Comment (tusharm-cuelogic) [inline bms/templates/registration/header_sidebar.html]: Why you have used <body> and <html> tags in this file it should be in base template and it should be extended in base template.

File: bms/bms_app/views.py [hunk 4]
```diff
@@ -49,7 +48,7 @@ def register(request):
                   'of the bugs which are as follows:\nAuthentication and Authorization,'
                   'Products, Bug, Attachment, Admin, Users, Configuration,'
                   'Log View, Search & View, '
-                  'Comments and tagging.\n\nLogin: https://www.facebook.com/ybts.ybts.9  '
+                  'Comments and tagging.\n\nLogin: http://172.21.31.90:8001  '
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py:51]: why you have used static URL?

**Pull request merged**